app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,39 @@
+class MessagesController < ApplicationController
+  def index
+    @messages = Message.all
+  end
+
+  def show
+    @message = Message.find(params[:id])
+  end
+
+  def new
+    @message = Message.new
+  end
+
+  def create
+    Message.create(message_params)
+    # message = Message.create!(message_params)
+    # redirect_to message
+  end
+
+  def edit
+    @message = Message.find(params[:id])
+  end
+
+  def update
+    message = Message.find(params[:id])
+    message.update(message_params)
+  end
+
+  def destroy
+    message = Message.find(params[:id])
+    message.destroy
+  end
+
+  private
+
+  def message_params
+    params.require(:message).permit(:name, :age)
```

Comment:
カラムの変更に対応できていません。

```suggestion
    params.require(:message).permit(:title, :content)
```

ビューのファイルも全く変更できていませんよ。